rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -53,13 +53,13 @@ rocalSequenceRearrange(
             THROW("sequence_length passed should be bigger than 0")
         auto input = static_cast<Image*>(p_input);
         auto info = ImageInfo(input->info().width(), input->info().height_single(),
-                              context->master_graph->internal_batch_size() * new_sequence_length,
+                              context->master_graph->user_batch_size() * new_sequence_length,
                               input->info().color_plane_count(),
                               context->master_graph->mem_type(),
                               input->info().color_format() );
         output = context->master_graph->create_image(info, is_output);
         std::shared_ptr<SequenceRearrangeNode> sequence_rearrange_node =  context->master_graph->add_node<SequenceRearrangeNode>({input}, {output});
-        sequence_rearrange_node->init(new_order, new_sequence_length, sequence_length, context->master_graph->internal_batch_size());
+        sequence_rearrange_node->init(new_order, new_sequence_length, sequence_length, context->master_graph->user_batch_size());
```

Comment:
Use the changes from this commit
https://github.com/SundarRajan28/MIVisionX/commit/6124b726db82b46139fdb138f8a499c66575ed11